### PR TITLE
Update sub menu item active style

### PR DESCRIPTION
**Pull request by nirzar**
The sub menu item active (is-on) style should be different than root item active style. 

This patch matches the spec on zeplin

### CODE REVIEW

File: css/wmui-style-guide.dev.css [hunk 1]
```diff
@@ -334,9 +334,17 @@ code {
 	border-left: 1px solid var( --wmui-color-base80 );
 	padding-left: 1.6rem;
 	font-size: 1.4rem;
+    border-radius: 0;
```
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: It should reset only on the beginning, so `border-radius: 0 var( --border-radius-base ) var( --border-radius-base ) 0;`

File: css/wmui-style-guide.dev.css [hunk 2]
```diff
@@ -334,9 +334,17 @@ code {
...
 	transition: background-color var( --transition-medium ), color var( --transition-medium );
 }
 
+.nav--main .nav__sub-item.is-on a {
+    background: #fff;
```
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: Needs to be variable `var( --background-color-primary )`.

File: css/wmui-style-guide.dev.css [hunk 3]
```diff
@@ -334,9 +334,17 @@ code {
...
+    color: var( --wmui-color-base0 );
+    font-weight: bold;
```
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: `font-weight` after `border` prop.

File: css/wmui-style-guide.dev.css [hunk 4]
```diff
@@ -334,9 +334,17 @@ code {
 	border-left: 1px solid var( --wmui-color-base80 );
 	padding-left: 1.6rem;
 	font-size: 1.4rem;
+    border-radius: 0;
 	transition: background-color var( --transition-medium ), color var( --transition-medium );
 }
 
+.nav--main .nav__sub-item.is-on a {
+    background: #fff;
+    color: var( --wmui-color-base0 );
```
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: Better to use semantical variable `--color-base--active`

File: css/wmui-style-guide.dev.css (hunk 1 above)
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: Also we're following a CSS property order as code guidelines: https://www.mediawiki.org/wiki/Manual_talk:Coding_conventions/CSS/Archive_1#CSS.2FLess_property_order_proposal `border-radius` should come after `border-left`.

**Comment by juliengirault** [comment]
Use tabs instead of spaces. Yeah I know... 😄

File: css/wmui-style-guide.css [hunk 5]
```diff
@@ -931,12 +931,20 @@ code {
...
+    -moz-border-radius: 0 2px 2px 0;
+         border-radius: 0 2px 2px 0;
 	padding-left: 1.6rem;
 	font-size: 1.4rem;
 	-webkit-transition: background-color 250ms, color 250ms;
 	-moz-transition: background-color 250ms, color 250ms;
 	transition: background-color 250ms, color 250ms;
 }
+.nav--main .nav__sub-item.is-on a {
+    background-color: #fff;
```
Comment (prtksxna) [inline css/wmui-style-guide.css]: Need tabs here?

File: css/wmui-style-guide.css [hunk 6]
```diff
@@ -931,12 +931,20 @@ code {
...
+    color: #000;
+    border-left-color: #000;
+    font-weight: bold;
```
Comment (prtksxna) [inline css/wmui-style-guide.css]: 700?

**Pull request merged**